### PR TITLE
debatable improvements

### PR DESCRIPTION
**Pull request by supakeen**
Doing these as separate commits as each can probably be discussed/removed from the PR separately. Most of these are likely opinionated as well.

### CODE REVIEW

File: how-does-it-work.md [hunk 1]
```diff
@@ -13,26 +13,26 @@ In fact it can be started as a typical container in [podman](http://podman.io/)
 The [bootc tool](https://github.com/containers/bootc) applies a container image as an update on an already running Linux system.
 The contents are written to the existing filesystem, switching out the /usr and /boot directories of the Linux system.
 One then reboots into the new behavior of the system.
-The new kernel in the bootable container image must of course support the filesystem, and storage devices on the system.
+The new kernel in the Bootable Container Image must of course support the filesystem, and storage devices on the system.
```
Comment (stefwalter) [inline how-does-it-work.md:16]: Why would we capitalize this one? I think this is just part of a sentence.

File: how-does-it-work.md [hunk 2]
```diff
@@ -13,26 +13,26 @@ In fact it can be started as a typical container in [podman](http://podman.io/)
...
 
 Using the boot loader it is possible to switch back to the previous operating system image, thus enabling rollback to the previous behavior of the system.
-Using this methodology, it’s expected to be able to start with a stock Linux system, and completely change the behavior of that system using the new bootable container image, and then iterate as with further images.
+Using this methodology, it’s expected to be able to start with a stock Linux system, and completely change the behavior of that system using the new Bootable Container Image , and then iterate as with further images.
```
Comment (stefwalter) [inline how-does-it-work.md:19]: Ditto here, I don't think this one should be capitalized.

File: index.md [hunk 3]
```diff
@@ -1,13 +1,13 @@
----
+-https://specs.opencontainers.org/image-spec/)--
```
Comment (stefwalter) [inline index.md:1]: Is this an accidental copy paste?

File: index.md [hunk 4]
```diff
@@ -1,13 +1,13 @@
...
 title: Goals
 layout: home
 ---
 
-## Image Based Linux with Bootable container images
+## Image Based Linux with Bootable Container Images
```
Comment (stefwalter) [inline index.md:6]: Here the capitalization makes sense, in a title.

**Review by stefwalter — CHANGES_REQUESTED** [review]
Thanks for the contribution. Lots of nice fix ups.

However, I don't think we would capitalize "bootable contaner images" and "image based linux" where ever they appear in the text. Agree we can capitalize in titles though.

Could you also add a ```Signed-off-by```? I'm adding a description of this in #4

File: index.md (hunk 3 above)
Comment (supakeen) [reply index.md:1]: It is, let me remove that.

Comment (supakeen) [reply index.md]: Done.

**Comment by supakeen** [comment]
I've backed out the capitalization commit and replaced it with one that only capitalizes in the title :)

**Review by stefwalter — APPROVED** [review]

**Pull request merged**